external-docs/docs/guides/configuration/templates.md

```diff
@@ -80,3 +80,89 @@ Units used, have to match the terminal part of the URI exactly including case se
 | Template                                                                       | Description                                    | Example                                                                              |
 |--------------------------------------------------------------------------------|------------------------------------------------|--------------------------------------------------------------------------------------|
 | [qudt-units](https://purl.org/csv-cubed/qube-config/templates/qudt-units.json) | A template which contains units of measurement | [http://qudt.org/vocab/unit/PoundSterling](http://qudt.org/vocab/unit/PoundSterling) |
+
+## How to deal with Mixed Periods
+
+The purpose of this section is to provide guidance on configuration of a data set with mixed time periods, e.g. daily, weekly, monthly, government-year, half-year, etc. Also, an example of how to wrangle a dataset using Python so that the CSV can be used to create a CSV-W is provided.
+
+In the table below we have two different types of dates, representing daily and weekly frequencies. The format of the daily figures is acceptable. However, there is an issue with the weekly data. The date `2019-04-01` represents the week beginning `2019-04-01`, and `2019-04-08` represents the following week. Instead, the ISO8601 format `week/YYYY-WWW` should be used, e.g., the week beginning `2019-04-01` should be `week/2019-W14`.
+
+| Period     | Frequency | ... |
+|------------|-----------|-----|
+| 2019-04-01 | Daily     | ... |
+| 2019-04-02 | Daily     | ... |
+| 2019-04-01 | Weekly    | ... |
+| 2019-04-08 | Weekly    | ... |
+
+**NB:** it is important to validate whether the weekly data matches the ISO8601 definition, as this is slightly different from the traditional Gregorian calendar.
```

Comment:
Can we link to somewhere which defined what the ISO8601 format is. Either add a term to our glossary or link to [here](https://www.iso.org/iso-8601-date-and-time-format.html).